File: 344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        i = 0;
        
        while(i < len(s) / 2):
            tmp = s[i]
            s[i] = s[len(s) - (1 + i)]
            s[len(s) - (1 + i)] = tmp
            i = i + 1

        """
        Do not return anything, modify s in-place instead.
        """
        

File: 344. Reverse String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) 
    {
        int i = 0;
        
        while(i < s.size() / 2)
        {
            char tmp = s[i];
            s[i] = s[s.size() - (1 + i)];
            s[s.size() - (1 + i)] = tmp;
            i++;
        }
    }
};